### ISSUE TITLE
Add visual image for the Image Alt Attributes automatically

### ISSUE THREAD

**Issue by sofianbettayeb** [die-Manufaktur/AI-SEO-Copilot-for-Webflow#278]
### Version

beta

### What happened?

Image Alt Attributes Analysis

https://screen.studio/share/xi3QzmEP

You can generate all missing alt text in one click. The alt text is generated by GPT by looking at the picture.
<img width="504" alt="Image" src="https://github.com/user-attachments/assets/4d9029ae-e932-41aa-8d87-2d773394d7dd" />

Once done, the user can 
edit the alt text
Insert the edited alt text
Insert all in one all the alt text

<img width="479" alt="Image" src="https://github.com/user-attachments/assets/43e5b66a-6192-45d2-b231-43f4fa2b8374" />

- Identifies images with missing alt text
- Identifies images with alt text that doesn't include the target keyphrase
- Provides suggestions for improved alt text that includes the keyphrase

### Relevant log output

```shell

```
